src/main/java/org/oasis_eu/spring/datacore/model/DCModel.java

```diff
@@ -95,6 +106,9 @@ public int hashCode() {
         @JsonProperty("dcmf:name")
         private String name;
 
+        @JsonProperty("dcmf:documentation")
+        private String description;
```

Comment:
it's clearer to keep the same naming : documentation

Reply:
OK